tutorials/ingest-real-time-websocket-data.md

```diff
@@ -138,18 +156,8 @@ websocket object and establish connection.
      'fails': None
     }
     ```
-
-</Procedure>
-
-<Highlight type="note">
-To keep the websocket connection alive indefinitely, use the `keep_alive()`
-function of the wrapper library. It makes sure the connection will
-stay active until it gets terminated. If you don't add this line the
-connection might break instantly.
-</Highlight>
-
 When you have established a connection to the websocket server,
-wait a few seconds, and you can see actual data records, like this:
+wait a few seconds, and you can see data records, like this:
 
 ```bash
```

Comment:
This code block isn't indented properly:

![image](https://github.com/timescale/docs/assets/3914967/3c42f0bc-c53f-43c9-a00d-a5bcdd973982)